app/src/ui/check-runs/ci-check-run-actions-logs.tsx

```diff
@@ -145,10 +151,7 @@ export class CICheckRunActionLogs extends React.PureComponent<
     isInGroup: boolean = false
   ): JSX.Element {
     const cn = classNames('line', lineData.className)
-    const style = isInGroup
-      ? // Meh... the +10 is due to the margin-left of the log group = var(--spacing).... if this changes oh no.. we broke - maybe can make this better in a future css improving pr.
-        { marginLeft: -1 * (this.logLineNumberWidth + 10) }
-      : undefined
+    const style = undefined
     return (
       <div className={cn} key={index} style={style}>
```

Comment:
You can get rid of `style`, right?
```suggestion
    return (
      <div className={cn} key={index}>
```